src/basilisp/lang/obj.py

```diff
@@ -97,9 +97,12 @@ def map_lrepr(
 
     kwargs = _process_kwargs(**kwargs)
 
+    kw_items = kwargs
+    kw_items["human_readable"] = False
```

Comment:
```suggestion
    kw_items = kwargs.copy()
    kw_items["human_readable"] = False
```

The existing code does not copy the value, only the reference. You need to use `dict.copy()` to get a shallow copy of the dictionary (shallow is fine here I think since these values are scalar only).

Reply:
Updated, thanks for the tip